src/index.html

```diff
@@ -11,7 +11,141 @@
+        <div class="form__field">
+          <label>
+            Surname:
+            <input type="text" name="surname">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Name:
+            <input type="text" name="name">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            How old are You?
+            <input type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Full date of birth:
+            <input type="date" name="birth-date">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            I accept the term of the agreement
+            <input type="checkbox" name="tos-box">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="section section2">
+        <legend>Registration</legend>
+        <div class="form__field">
+          <label>
+            E-mail:
+            <input type="email"
+              name="email-addr"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            Password:
+            <input type="password"
+              name="password"
+              minlength="5"
+              maxlenght="12"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="section section3">
+        <legend>An interesting fact about you!</legend>
+        <div class="form__field">
+          <label>
+            Do you love cats?
+            <input type="radio"
+              name="cats"
+              value="Yes"
+            >
+              Yes
+          </label>
+          <label>
+            <input type="radio"
+              name="cats"
+              value="No"
+            >
+              No
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            What is your favorite color?
+            <input type="color" name="color">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            What time do you go to bed?
+            <input type="time" name="time">
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            What are your favorite brand of cars?
+            <select name="car" multiple>
+              <option value="bmw">BMW</option>
+              <option value="audi">Audi</option>
+              <option value="lada">Lada</option>
+            </select>
+          </label>
+        </div>
+        <div class="form__field">
+          <label>
+            How do you rate our work?
+            <input type="range"
+              name="rate"
+            >
```

Comment:
let's define the range as well (min, max)
```suggestion
            <input
              type="range"
              name="rate"
            >
```